sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -700,12 +700,7 @@ class VersionsSuite extends SparkFunSuite with Logging {
 
     test(s"$version: SPARK-17920: Insert into/overwrite avro table") {
```

Comment:
BTW, would you guys mind if I ask why the tests are different between master and branch-2.2? Just curious.

Reply:
In master, this testcase uses an avro file & decimal schema from [https://github.com/apache/spark/pull/19003], but in branch-2.2  that  PR 19003 is not backported. So I had to change the testcase in branch-2.2 to avoid using  schema & data added by PR 19003.
If we are fine to  backport  the PR 19003 in branch-2.2, same testcase in master can be used in branch-2.2 too. Please give your suggestions.

Reply:
Ah, OK if there is a reason. I was just curious.